### PR TITLE
Add Apache license to a few files

### PR DESCRIPTION
**Pull request by ash211**
Required for `./dev/check-license` to succeed

### CODE REVIEW

File: resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager [hunk 1]
```diff
@@ -1,2 +1,18 @@
+#
```
Comment (foxish) [inline resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager]: Doesn't seem like other META-INF directories under spark-sql or ml-lib have this header.

Comment (ash211) [reply resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager]: Maybe they have an exclusion that ignores license check on that META-INF folder but not this one?  Would you like to match that pattern?

Comment (foxish) [reply resource-managers/kubernetes/core/src/main/resources/META-INF/services/org.apache.spark.deploy.rest.kubernetes.DriverServiceManager]: For consistency, that seems like a good idea to match that

**Comment by mccheah** [comment]
Looks fine. This should also be cherry-picked onto our release branch, right?

**Pull request merged**